### PR TITLE
Add pipeline schedule API

### PR DESCRIPTION
**Pull request by Lorac**

### CODE REVIEW

File: pipeline_schedules.go [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+// https://docs.gitlab.com/ce/api/pipeline_schedules.html#edit-a-pipeline-schedule-variable
+type EditPipelineScheduleVariableOptions struct {
+	Value *string `url:"value" json:"value"`
+}
+
+// EditPipelineScheduleVariable creates a pipeline schedule variable.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/pipeline_schedules.html#edit-a-pipeline-schedule-variable
+func (s *PipelineSchedulesService) EditPipelineScheduleVariable(pid interface{}, schedule, key int, opt *EditPipelineScheduleVariableOptions, options ...OptionFunc) (*PipelineVariable, *Response, error) {
```
Comment (svanharmelen) [inline pipeline_schedules.go]: `key` should be a `string`

File: pipeline_schedules.go [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+	}
+
+	return p, resp, err
+}
+
+// DeletePipelineScheduleVariable creates a pipeline schedule variable.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/pipeline_schedules.html#delete-a-pipeline-schedule-variable
+func (s *PipelineSchedulesService) DeletePipelineScheduleVariable(pid interface{}, schedule, key int, options ...OptionFunc) (*PipelineVariable, *Response, error) {
```
Comment (svanharmelen) [inline pipeline_schedules.go]: `key` should be a `string`

**Comment by Lorac** [comment]
@svanharmelen All fixed :)

**Comment by svanharmelen** [comment]
Thanks! Looks great now 👍

**Pull request merged**